### PR TITLE
Test for both JOB and UDF

### PR DESCRIPTION
**Pull request by uellue**
Make sure that the functionality is tested with the legacy JOB back-end
and the new UDF back-end for the time being

## Contributor Checklist:

* [x] I have added myself to [the creators.json file](https://github.com/LiberTEM/LiberTEM/blob/master/packaging/creators.json)
* [x] I have added/updated test cases

### CODE REVIEW

File: tests/io/test_empad.py [hunk 1]
```diff
@@ -124,15 +128,19 @@ def test_sum_analysis(default_empad, lt_ctx):
     assert np.count_nonzero(results[0].raw_data) > 0
 
 
-def test_pick_job(default_empad, lt_ctx):
+def test_pick_job(default_empad, lt_ctx, TYPE):
```
Comment (sk1p) [inline tests/io/test_empad.py]: Missing the parametrize mark here?

Comment (uellue) [reply tests/io/test_empad.py]: Actually, added the TYPE in error since this is using the JOB directly. No UDF here!

Nice catch, thx!

**Comment by uellue** [comment]
@sk1p Did you test it with your file collection, by the way? I couldn't run many of the tests on my system.

**Comment by sk1p** [comment]
> @sk1p Did you test it with your file collection, by the way? I couldn't run many of the tests on my system.

I didn't. That's actually something I want to fix - I guess I'll devise an interim solution for #86 ASAP.

**Comment by uellue** [comment]
> > @sk1p Did you test it with your file collection, by the way? I couldn't run many of the tests on my system.
>
> I didn't. That's actually something I want to fix - I guess I'll devise an interim solution for #86 ASAP.

OK, then this PR should probably wait until the new infrastructure is ready.

**Comment by sk1p** [comment]
I'll merge this now and then we'll deal with the fallout later. Thanks!

**Pull request merged**